### ISSUE TITLE
Test localhost:2304/tweak failed

### ISSUE THREAD

**Issue by bschuele** [ChristophSchranz/Octoprint-PrePrintService#5]
My first test failed with this log message

```
[2020-08-12 05:05:52,218] INFO in tweak-service: Using Tweaker actions: tweak
[2020-08-12 05:05:52,218] INFO in tweak-service: Running Tweak with command: 'python3 /src/Tweaker-3/Tweaker.py -i /src/uploads/3DBenchy.stl -x -vol -t binarystl -o /src/uploads/tweaked_3DBenchy.stl'
usage: Tweaker.py [-h] [-i  INPUTFILE] [-o  OUTPUTFILE] [-vb] [-p] [-c]
                  [-t  OUTPUT_TYPE] [-x] [-v] [-r] [-fs FAVSIDE]
                  [-min MINIMIZE]
Tweaker.py: error: unrecognized arguments: -vol
```

**Comment by ChristophSchranz** [comment]
The parameter `-vol` isn't supported anymore, just delete it in the source file and rerun again.

**Comment by ChristophSchranz** [comment]
This issue was fixed in commit 4d3a64f070ba9ebc48404704313988ee899225f6.
